opencog/reasoning/RuleEngine/rule-engine-src/InferenceSCM.cc

```diff
@@ -55,8 +59,28 @@ Handle InferenceSCM::do_forward_chaining(Handle h) {
 	AtomSpace *as = SchemeSmob::ss_get_env_as("cog-fc");
 	ForwardChainer fc(as);
 	fc.do_chain(h); //START FORWARD CHAINING
-    HandleSeq result = fc.get_chaining_result();
-    return as->addLink(LIST_LINK,result,TruthValue::DEFAULT_TV());
+	HandleSeq result = fc.get_chaining_result();
+	return as->addLink(LIST_LINK, result, TruthValue::DEFAULT_TV());
```

Comment:
trivial comment: you don;t have to say "TruthValue::DEFAULT_TV()" because if you don't specify it, it will become the default anyway...